### ISSUE TITLE
Suggestion: Make number of QSO's shown in top mode a runtime option

### ISSUE THREAD

**Issue by khaytsus** [kencormack/pistar-lastqso#4]
If one were replacing the web UI for this, they'd want to see more than the most 5 recent QSO's.  Can you add a runtime parameter to show more QSOs in the list?

**Comment by kencormack** [comment]
I'd thought about it shortly after I started writing this, and realized that screens only have so many lines to work with,  I discovered this even before I added the "-t|--top" feature.  I wanted the emphasis to be on the additional data that the script was meant to provide (Caller Names, TG names, Talker Alias, QTH, etc.), that ISN'T available in the GUI.  Admittedly, the large font stuff eats into the screen lines as well, but that came out of my desire to see what's going on from across the room.  I did, however, add a way to disable that and give back more room to the QSOs. 

On a wide display (as most screens are these days, I also wished that the most common console, xterm, and VT100-type terminals allowed for split-screens, such that the script as you see it today would run in the left side of the screen, while the right half was nothing but history.  But alas, those term-types do not support such "windowing management" features.  (And trying to programmatically account for accurate cursor placement on two independently scrolling halves of a single screen would be asinine to consider, in a shell script.

My concession to having any history at all, was to limit it to 5 lines in order to maximize screen-lines available for the QSO data.  I don't see that changing.
